src/adv.py

```diff
@@ -49,3 +55,63 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+while True:
+    current_room = player.room
+    player_inventory = [x.name for x in player.inventory] if len(
+        player.inventory) else "Empty"
+    room_inventory = [x.name for x in current_room.item] if len(
+        current_room.item) else "Empty Room"
+
+    print(
+        f" === Entered {player.room.room_name}, {current_room.room_description} ===")
+    print(f"*You have found a {room_inventory}")
+
+    print(f"Inventory: [{len(player.inventory)}]")
+    print(f"Bag: {player_inventory}")
+
+    move = input(
+        '[Movement]: [N]/[S]/[W]/[E]. [Take/Drop]: [T]/[D]. [Q]-Quit \n :> ')
+
+    action = move[0:4]
+    item = move[5:]
+
+    if move.lower() == 'n':
+        if current_room.n_to is not None:
+            player.room = current_room.n_to
+            print(f"\t==> Moving north")
+        else:
+            print("XXX Can't go that way XXX")
+    elif move.lower() == 's':
+        if current_room.s_to is not None:
+            player.room = current_room.s_to
+            print(f"\t==> Moving south")
+        else:
+            print("XXX Can't go that way XXX")
+    elif move.lower() == 'e':
+        if current_room.e_to is not None:
+            player.room = current_room.e_to
+            print(f"\t==> Moving east")
+        else:
+            print("XXX Can't go that way XXX")
+    elif move.lower() == 'w':
+        if current_room.w_to is not None:
+            player.room = current_room.w_to
+            print(f"\t==> Moving West")
+        else:
+            print("XXX Can't go that way XXX")
```

Comment:
It would be nicer to have this logic factored out to the Player class and call that class method inside adv.py file